azurerm/resource_arm_kubernetes_cluster_test.go

```diff
@@ -220,6 +220,31 @@ func TestAccAzureRMKubernetesCluster_internalNetwork(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMKubernetesCluster_internalNetworkRouteTable(t *testing.T) {
```

Comment:
we should be able to combine this into the `TestAccAzureRMKubernetesCluster_advancedNetworking[X]Complete` test